release_notes/ocp_3_9_release_notes.adoc

```diff
@@ -0,0 +1,434 @@
+== About This Release
+
+Red Hat {product-title} version 3.9
+(link:https://access.redhat.com/errata/RHSA-2018:XXXX[RHSA-2018:XXXX]) is now
+available. This release is based on
+link:https://github.com/openshift/origin/releases/tag/v3.7.0-rc.0[OpenShift
+Origin 3.9]. New features, changes, bug fixes, and known issues that pertain to
+{product-title} 3.9 are included in this topic.
+
+{product-title} 3.9 is supported on RHEL 7.3, 7.4.2, and Atomic Host 7.4.2 and
+newer with the latest packages from Extras, including Docker 1.12.
+
+For initial installations, see the
+xref:../install_config/install/planning.adoc#install-config-install-planning[Installing
+a Cluster] topics in the
+xref:../install_config/index.adoc#install-config-index[Installation and
+Configuration] documentation.
+
+To upgrade to this release from a previous version, see the
+xref:../install_config/upgrading/index.adoc#install-config-upgrading-index[Upgrading
+a Cluster] topics in the
+xref:../install_config/index.adoc#install-config-index[Installation and
+Configuration] documentation.
+
+[[ocp-39-new-features-and-enhancements]]
+== New Features and Enhancements
+
+This release adds improvements related to the following components and concepts.
+
+[[ocp-39-container-orchestration]]
+=== Container Orchestration
+
+[[ocp-39-crio]]
+==== CRI-O
+
+CRI-O v1.0 is a lightweight, native Kubernetes container runtime interface. By
+design, it provides only the runtime capabilities needed by the kubelet. CRI-O is
+designed to be part of Kubernetes and evolve in lock-step with the platform.
+
+CRI-O brings:
+
+* A minimal and secure architecture.
+* Excellent scale and performance.
+* The ability to run any Open Container Initiative (OCI) or docker image.
+* Familiar operational tooling and commands.
+
+image::crio-3-7.png[CRI-O]
+
+To install and run CRI-O alongside `docker`, set the following in the
+`[OSEv3:vars]` section
+xref:../install_config/install/advanced_install.adoc#configuring-ansible[Ansible inventory file] during cluster installation:
+
+----
+openshift_use_crio=true
+----
+
+This setting pulls the *openshift3/cri-o* system container image from the
+link:https://access.redhat.com/containers[Red Hat Registry] by default. If you
+want to use an alternative CRI-O system container image from another registry,
+you can also override the default using the following variable:
+
+----
+openshift_crio_systemcontainer_image_override=<registry>/<repo>/<image>:<tag>
+----
+
+[NOTE]
+====
+The `atomic-openshift-node` service must be RPM- or system container-based when
+using CRI-O; it cannot be `docker` container-based. The installer protects again
+using CRI-O with `docker` container nodes and will halt installation if
+detected.
+====
+
+When CRI-O use is enabled, it is installed alongside `docker`, which currently
+is required to perform build and push operations to the reigstry. Over time,
+temporary `docker` builds can accumulate on nodes. You can optionally set the
```

Comment:
How can I limit the nodes that docker is installed on with the installer? 
@sdodson

Reply:
Just set openshift_use_crio=true only on the nodes you wish to use crio.

Reply:
docker is still required on all nodes at this time though.

Reply:
I was under the impression that docker was only needed on nodes that executed builds (which we can limit to some nodes).

Reply:
> I was under the impression that docker was only needed on nodes that executed builds (which we can limit to some nodes).

This is true, builds and pushes, but I believe @sdodson's assumption is that you'd want to be able to build on all nodes.